internal/build/phases.go

```diff
@@ -183,31 +187,22 @@ func (l *Lifecycle) newExport(repoName, runImage string, publish bool, launchCac
 		)
 
 		return phaseFactory.New(configProvider), nil
-	}
+	} else {
```

Comment:
I can see why you added this... this `else` would've likely alerted us all to the initial problem. But now the linter is unhappy :/